source/extensions/filters/http/jwt_authn/verifier.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "envoy/common/pure.h"
+#include "envoy/config/filter/http/jwt_authn/v2alpha/config.pb.h"
+
+#include "jwt_verify_lib/status.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace JwtAuthn {
+
+class Verifier;
+typedef std::shared_ptr<const Verifier> VerifierSharedConstPtr;
+
+/**
+ *  Verifies JWT based on the requirement rules.
+ */
+
+class Verifier {
+public:
+  virtual ~Verifier() {}
+  virtual ::google::jwt_verify::Status verify(const std::vector<std::string>& issuers) const PURE;
```

Comment:
per google style guide: <https://google.github.io/styleguide/cppguide.html>
please add comment for each function in the header file.